operations/research-and-development/product/how-to-guides-for-product/release-announcement.md

```diff
@@ -0,0 +1,58 @@
+# How to Write a Feature Release Announcement
+
+This document outlines how to write a release announcement and release tweets for a feature. These are used at the release in our release newsletter and Twitter. 
+
+Each Product Manager is responsible for writing a release announcement for every feature, integration or announcement they want included in the release newsletter.
+
+[Here is an example of a previous completed release announcement.](https://docs.google.com/document/d/1BqgLFqvpYJFm6TfrqkCUyYIs666BL1nawpgw6a4CpGA/edit?usp=sharing)
+
+## Overview
+
+It is important to think of the Release Announcement as a whole, as a press release.  We are communicating changes to our current and prospective customers so we want it to be exciting as well as informative.  The Release Announcement will be a coordinated document, with each product manager contributing their own features as sub-headlines.  The headline and first paragraph will be written by PM leads and the CPO after all other features have been provided. 
+
+*Note: Consider using this information in ProductBoard for your features to validate the messaging and to keep consistency across channels.* 
+
+### Timeline
+A release announcement for each feature should be started early in the product planning process. Each Product Manager is directly responsible for writing the announcement for each of their features. For large product features or new products, a PRFAQ should be completed. For smaller features, a “mini-PRFAQ” should be completed in the form of a blog post. 
+
+Release announcements and tweets are due from Product Managers at T-14, so a first draft can be completed by T-11, per our [release announcement process](operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements). They should be provided to your Lead (Lindsay or Katie), who will coordinate all into one single document for the release. A final draft of the combined release annoucenment for the newsletter will be provided for review to CPO and MLT team by T-10. Finally, the approved release annoucenment will be provided to marketing by T-2 and published on release day. 
+
+### Feature Headline
+
+The headline for your feature should start with a verb and describe what the feature allows customer to do and the edition it is provided in. 
+
+Example: ``Sync Office 365 Calendar status with Mattermost (E20 edition)``
+
+### Feature Benefit
+
+A quick summary sentence should follow with the benefit the feature provides. 
+
+Example: ``Enable your users to get more control over their schedule without leaving Mattermost. ``
+
+### Feature Details
+
+Next, describe in more detail how the feature works and what functionality is included.  This could take the form of a bulleted list or a few more sentences.  
```

Comment:
```suggestion
Next, describe in more detail how the feature works and what functionality is included. This could take the form of a bulleted list or a few more sentences. 
```